packages/create-emotion/README.md

```diff
@@ -39,7 +39,7 @@ export const {
 
 - Using emotion in embedded contexts such as an `<iframe/>`
 
-- Setting a [nonce](/packages/cache#nonce-string) on any `<style/>` tag emotion creates for security purposes
```

Comment:
problem is that we'd like those URLs to work on GitHub as well. Those URLs are rewritten when building site which is I think is happening here - https://github.com/emotion-js/emotion/blob/d0b2a94ab9d5648667447dbd78e7a2e3e93de42a/site/plugins/gatsby-remark-fix-links/index.js . This script has to have some bug in it. I think I've tried to fix this in the past, just very briefly though - couldn't find more time to finish the work. My attempt can be found here https://github.com/emotion-js/emotion/pull/1432/files . Maybe you would like to give it a go?

Reply:
Will do. 

Do you think the fact that the hash-link in the docs says `nonce-string`, and the correct hash is `nonce` could be the culprit?

Reply:
It should be corrected as well, but its not the main culprit i think. The main one being this broken rewrite script.